tests/Moq.Analyzers.Test/Helpers/ConfigAnalyzerVerifier.cs

```diff
@@ -0,0 +1,29 @@
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+
+namespace Moq.Analyzers.Test.Helpers;
+
+/// <summary>
+/// A specialized analyzer verifier for testing scenarios where analyzers are disabled
+/// via configuration files like .editorconfig. This helper encapsulates the complexity
+/// of setting up Test&lt;&gt; instances with configuration files.
+/// </summary>
+internal static class ConfigAnalyzerVerifier<TAnalyzer>
```

Comment:
This is duplicative of the `AnalyzerVerifier` class. Determine how to extend that with this functionality rather than duplicating to keep things DRY